utils/ExperienceAnalyser.js

```diff
@@ -0,0 +1,31 @@
+export default function ExperienceAnalyser(data) {
```

Comment:
the naming convention for normal JavaScript functions is camelCase, I would therefore rename this function to be:

```suggestion
export default function experienceAnalyser(data) {
```